app/public/database/Gemorskos.sql

```diff
@@ -1,45 +1,85 @@
+--
+-- Tabelstructuur voor tabel `Evenement`
+-- Met indexen voor de tabel
+--
+DROP TABLE IF EXISTS `Evenement`;
+CREATE TABLE `Evenement` (
+  `evenement_id` int AUTO_INCREMENT NOT NULL,
+  `evenement_naam` varchar(40) NOT NULL,
+  `beschrijving` text NOT NULL,
+  `dag` date DEFAULT NULL,
+  `tijd` time DEFAULT NULL,
+  `straatnaam` varchar(26) NOT NULL,
+  `stad` varchar(40) NOT NULL,
+  `postcode` varchar(6) NOT NULL,
+  PRIMARY KEY (`evenement_id`)
 );
-
-CREATE TABLE `Evenement_Detail`(
-`redacteur_id` INT,
-`journalist_id` INT,
-`fotograaf_id` INT,
-`evenement_id`INT NOT NULL,
-FOREIGN KEY (`redacteur_id`) REFERENCES `Medewerkers`(`medewerker_id`) ON UPDATE CASCADE ON DELETE NO ACTION,
-FOREIGN KEY(`journalist_id`) REFERENCES `Medewerkers`(`medewerker_id`) ON UPDATE CASCADE ON DELETE NO ACTION,
-FOREIGN KEY(`fotograaf_id`) REFERENCES `Medewerkers`(`medewerker_id`) ON UPDATE CASCADE ON DELETE NO ACTION,
-FOREIGN KEY(`evenement_id`) REFERENCES `Evenement`(`evenement_id`) ON UPDATE CASCADE ON DELETE NO ACTION
+--
+-- Tabelstructuur voor tabel `Medewerkers`
+-- Met indexen voor de tabel
+--
+DROP TABLE IF EXISTS `Medewerkers`;
+CREATE TABLE `Medewerkers` (
+  `medewerker_id` int AUTO_INCREMENT NOT NULL,
+  `werk_functie_id` int(11) NOT NULL,
+  `voornaam` varchar(25) NOT NULL,
+  `achternaam` varchar(25) NOT NULL,
+  `email` varchar(55) NOT NULL,
+  `telefoonnummer` varchar(10) NOT NULL,
+  `wachtwoord` varchar(60) NOT NULL,
+  PRIMARY KEY (`medewerker_id`),
+  FOREIGN KEY (`werk_functie_id`) REFERENCES `Werk_Functie` (`werk_functie_id`) ON DELETE NO ACTION ON UPDATE CASCADE
 );
-
-CREATE TABLE `Bestand`(
-`bestand_id` INT AUTO_INCREMENT NOT NULL,
-`medewerker_id` INT NOT NULL,
-`evenement_id` INT NOT NULL,
-`bestandsnaam` VARCHAR(40),
-`bestand_grootte_MB` INT NOT NULL,
-`bestand_type` VARCHAR(10) NOT NULL,
-`upload_datum` DATE,
-`beschrijving` TEXT NOT NULL,
-PRIMARY KEY(`bestand_id`),
-FOREIGN KEY(`medewerker_id`) REFERENCES `Medewerkers`(`medewerker_id`) ON UPDATE CASCADE ON DELETE NO ACTION,
-FOREIGN KEY(`evenement_id`) REFERENCES `Evenement`(`evenement_id`) ON UPDATE CASCADE ON DELETE NO ACTION
+--
+-- Tabelstructuur voor tabel `Bestand`
+-- Met indexen voor de tabel
+--
+DROP TABLE IF EXISTS `Bestand`;
+CREATE TABLE `Bestand` (
+  `bestand_id` int AUTO_INCREMENT NOT NULL,
+  `medewerker_id` int(11) NOT NULL,
+  `evenement_id` int(11) NOT NULL,
+  `bestandsnaam` varchar(40) DEFAULT NULL,
+  `bestand_grootte_MB` int(11) NOT NULL,
+  `bestand_type` varchar(10) NOT NULL,
+  `upload_datum` date DEFAULT NULL,
+  `beschrijving` text NOT NULL,
+  PRIMARY KEY (`bestand_id`),
+  FOREIGN KEY (`medewerker_id`) REFERENCES `Medewerkers` (`medewerker_id`) ON DELETE NO ACTION ON UPDATE CASCADE,
+  FOREIGN KEY (`evenement_id`) REFERENCES `Evenement` (`evenement_id`) ON DELETE NO ACTION ON UPDATE CASCADE
+);
+--
+-- Tabelstructuur voor tabel `Evenement_Detail`
+-- Met indexen voor de tabel
+--
+DROP TABLE IF EXISTS `Event_Detail`;
```

Comment:
Isn't this supposed to be `Evenement_Detail` instead of `Event_Detail`?

Reply:
The table had to be named 'Evenement_Detail'.

Thanks for spotting the (minor) mistake.